_posts/pl/2022-11-08-jaxb.md

```diff
@@ -0,0 +1,53 @@
+---
+layout:    post
+title:     "Czy wiesz jak poprawnie tworzyć kontekst JAXB i instancję Marshallera/Unmarshallera?"
+date:      2022-11-08 06:00:00 +0100
+published: true
+didyouknow: true
+lang: pl
+author:    jwilczewski
+image:     /assets/img/posts/2022-11-08-jaxb/jaxb.jpg
+tags:
+
+- jaxb
+- java
+- marshaller
+- unmarshaller
+description: W naszych projektach często używamy JAXB w celu mapowania danych przesyłanych kopertach SOAP-owych na obiekty javowe. W tym wpisie znajdziesz wskazówki jak poprawnie wykonać takie mapowania.
+
+---
+
+W naszych projektach często używamy JAXB w celu mapowania danych przesyłanych kopertach SOAP-owych na obiekty javowe. Aby wykonać takie mapowanie należy:
+
+Utworzyć kontekst JAXB
+
+```java
+JAXBContext jaxbContext = JAXBContext.newInstance(User.class);
+```
+Utworzyć marshaller/unmarshaller (w zależności do tego czy  mapujemy obiekt javy na xml, czy w drugą stronę):
```

Comment:
czy  ma -> czy ma (chociaż to i tak pewnie się wyświetli z jedną spacją)

Reply:
done